install/installer/pkg/common/common.go

```diff
@@ -112,7 +112,7 @@ func DefaultEnv(cfg *config.Config) []corev1.EnvVar {
 	)
 }
 
-func WorkspaceTracingEnv(context *RenderContext) (res []corev1.EnvVar) {
+func WorkspaceTracingEnv(context *RenderContext, component string) (res []corev1.EnvVar) {
```

Comment:
Do we need this indirection here if we're actually keeping them the same? Could we instead consolidate on `func TracingEnv(...)`. If the need for these to actually be different arises, we can refactor.

Reply:
I don't know the reason for this structure.
ping @MrSimonEmms

Reply:
Nothing to do with me I'm afraid. This was added by @andrew-farries around 5 months ago - https://github.com/gitpod-io/gitpod/commit/1119e5578bfe502ae99b90708649ee19523c8cc0